examples/flutter_login/lib/login/bloc/login_bloc.dart

```diff
@@ -27,7 +27,6 @@ class LoginBloc extends Bloc<LoginEvent, LoginState> {
     emit(
       state.copyWith(
         username: username,
-        status: Formz.validate([state.password, username]),
```

Comment:
Can you explain please why this is no longer necessary?

Reply:
In new version, formz package did breaking changes by **decouple purity, validity, and submission status**. There also rename `FormzStatus` to `FormzSubmissionStatus`. Property status inside LoginState from now on will only reflect on submission status. On the other hand, validity and purity will be accessible via getter inside LoginState after implement `FormzMixin`.

Since `Formz.validate()` will return a boolean while status is FormzSubmissionStatus and accessible via getter. We don't need to refresh status property. We can access it on demand via `state.isValid`.